openrtb_ext/bidders.go

```diff
@@ -115,116 +128,130 @@ const (
-	"smartrtb":          BidderSmartRTB,
-	"smartyads":         BidderSmartyAds,
-	"somoaudience":      BidderSomoaudience,
-	"sonobi":            BidderSonobi,
-	"sovrn":             BidderSovrn,
-	"synacormedia":      BidderSynacormedia,
-	"tappx":             BidderTappx,
-	"telaria":           BidderTelaria,
-	"triplelift":        BidderTriplelift,
-	"triplelift_native": BidderTripleliftNative,
-	"ucfunnel":          BidderUcfunnel,
-	"unruly":            BidderUnruly,
-	"valueimpression":   BidderValueImpression,
-	"verizonmedia":      BidderVerizonMedia,
-	"visx":              BidderVisx,
-	"vrtcal":            BidderVrtcal,
-	"yeahmobi":          BidderYeahmobi,
-	"yieldlab":          BidderYieldlab,
-	"yieldmo":           BidderYieldmo,
-	"yieldone":          BidderYieldone,
-	"zeroclickfraud":    BidderZeroClickFraud,
+// BidderNames returns all core bidders.
+func BidderNames() []BidderName {
+	return []BidderName{
+		Bidder33Across,
+		BidderAdform,
+		BidderAdgeneration,
+		BidderAdhese,
+		BidderAdkernel,
+		BidderAdkernelAdn,
+		BidderAdman,
+		BidderAdmixer,
+		BidderAdOcean,
+		BidderAdoppler,
+		BidderAdpone,
+		BidderAdprime,
+		BidderAdtarget,
+		BidderAdtelligent,
+		BidderAdvangelists,
+		BidderAJA,
+		BidderApplogy,
+		BidderAppnexus,
+		BidderAudienceNetwork,
+		BidderAvocet,
+		BidderBeachfront,
+		BidderBeintoo,
+		BidderBetween,
+		BidderBrightroll,
+		BidderColossus,
+		BidderConnectAd,
+		BidderConsumable,
+		BidderConversant,
+		BidderCpmstar,
+		BidderDatablocks,
+		BidderDmx,
+		BidderEmxDigital,
+		BidderEngageBDR,
+		BidderEPlanning,
+		BidderGamma,
+		BidderGamoshi,
+		BidderGrid,
+		BidderGumGum,
+		BidderImprovedigital,
+		BidderInMobi,
+		BidderInvibes,
+		BidderIx,
+		BidderKidoz,
+		BidderKrushmedia,
+		BidderKubient,
+		BidderLifestreet,
+		BidderLockerDome,
+		BidderLogicad,
+		BidderLunaMedia,
+		BidderMarsmedia,
+		BidderMgid,
+		BidderMobileFuse,
+		BidderNanoInteractive,
+		BidderNinthDecimal,
+		BidderOpenx,
+		BidderOrbidder,
+		BidderPubmatic,
+		BidderPubnative,
+		BidderPulsepoint,
+		BidderRhythmone,
+		BidderRTBHouse,
+		BidderRubicon,
+		BidderSharethrough,
+		BidderSmaato,
+		BidderSmartAdserver,
+		BidderSmartRTB,
+		BidderSmartyAds,
+		BidderSomoaudience,
+		BidderSonobi,
+		BidderSovrn,
+		BidderSynacormedia,
+		BidderTappx,
+		BidderTelaria,
+		BidderTriplelift,
+		BidderTripleliftNative,
+		BidderUcfunnel,
+		BidderUnruly,
+		BidderValueImpression,
+		BidderVerizonMedia,
+		BidderVisx,
+		BidderVrtcal,
+		BidderYeahmobi,
+		BidderYieldlab,
+		BidderYieldmo,
+		BidderYieldone,
+		BidderZeroClickFraud,
+	}
 }
 
-// BidderList returns the values of the BidderMap
-func BidderList() []BidderName {
-	bidders := make([]BidderName, 0, len(BidderMap))
-	for _, value := range BidderMap {
-		bidders = append(bidders, value)
+// BuildBidderNameLookup builds a map of string to BidderName, to remain compatbile with the
+// prebioud BidderMap variable.
+func BuildBidderNameLookup() map[string]BidderName {
```

Comment:
Builds bidderMap in realtime now. I changed the name to try to make it clearer, but I'm not sure I succeeded. Naming suggestions here?